scripts/snippets/inf-hosp.r

```diff
@@ -0,0 +1,19 @@
+## first, choose random delays
+inf_hosp <- infection_times |>
+  mutate(
+    incubation_period = rgamma(n(), shape = 5, rate = 1),
+    onset_to_hosp = rlnorm(n(), meanlog = 1, sdlog = 1),
+    infection_date = as.Date(start_date + infection_time),
```

Comment:
if start_date is a date then this data won'
t have double censoring (as the input is discrete).

Reply:
(this is why we used a continuous model to simulate infections in the delay dist long paper)